CICD/github/launch_karate_tests.sh

```
@@ -20,7 +20,7 @@ cd bin
 ./startOpfab.sh prod
 #Logs for debug
 docker logs greenmail
-cd ../test/api/karate
+cd ../tests/api/karate
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)